projects/addon-mobile/components/sheet/directives/sheet-close/sheet-close.directive.ts

```diff
@@ -11,6 +11,15 @@ import {TUI_SHEET_DRAGGED, TUI_SHEET_SCROLL} from '../../sheet-tokens';
     selector: 'tui-sheet[close]',
 })
 export class TuiSheetCloseDirective {
+    private readonly zone = inject(NgZone);
+    // TODO: why TUI_SHEET_DRAGGED returned boolean?
+    private readonly dragged$ = inject<Observable<boolean>>(TUI_SHEET_DRAGGED);
```

Comment:
Let fix this mistake 
https://github.com/taiga-family/taiga-ui/blob/7b824fe415f732f865895c94632a0ace9d11d485/projects/addon-mobile/components/sheet/sheet-tokens.ts#L18

Reply:
fixed